deps.edn

```diff
@@ -901,15 +901,17 @@
   ;; https://github.com/borkdude/clj-kondo/
   :lint/clj-kondo
   {:extra-deps {clj-kondo/clj-kondo {:mvn/version "2021.12.16"}}
-   :main-opts  ["-m" "clj-kondo.main" "--lint" "src"]}
+   :main-opts  ["-m" "clj-kondo.main" "--lint" "src" "test"]}
 
 
   ;; Eastwood - classic lint tool
   ;; https://github.com/jonase/eastwood#running-eastwood-in-a-repl
   :lint/eastwood
   {:extra-deps {jonase/eastwood {:mvn/version "1.0.0"}}
-   :main-opts  ["-m" "eastwood.lint"
-                "{:source-paths,[\"src\"],:test-paths,[\"test\"]}"]}
+   ;; NOTE: source/test paths are accurately inferred, particularly for deps.edn projects,
```

Comment:
If such a note is needed, a shorter version would be preferable 
;; Eastwood accurately infers class paths, so are not explicitly required